src/index.html

```diff
@@ -11,6 +11,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action=""
+      method="post"
+      data-qa="keypress"
+    >
+      <div class="search-bar1 search-bar">
+        <input data-qa="big" class="search-bar__input bigInput"
+          type="search" name="bar1"
+          id="bar1" placeholder="Try &#34;Los Angeles&#34;">
```

Comment:
Please pay attention to my comments, remember about proper formatting
```suggestion
        <input 
        	data-qa="big" 
        	class="search-bar__input bigInput"
          type="search" 
          name="bar1"
          id="bar1" 
          placeholder="Try &#34;Los Angeles&#34;"
        >
```